charts/daemonset-app/templates/sealedsecret.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and (.Values.secrets) (eq .Values.secretsEngine "sealed") }}
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  name: {{ include "nd-common.fullname" . }}-secrets
+  annotations:
+    {{- /*
+    Always (for now) set scope to be namespace-wide. At any time if usecases arise to
+    for cluster-wide scope, we can update these template to make it configurable (this
+    would also mean a separation of SealedSecret objects if mixing-and-matching)
+    */}}
+    sealedsecrets.bitnami.com/namespace-wide: "true"
```

Comment:
I thought this was purely used for encrypting the value - how does this get used in decryption?

Reply:
This is very true and a good point/question. However, since we generate the ciphertext (encrypted blob) using kubeseal's `--raw` option (only for simpler end-user usability), we need to add the annotation to the sealedsecret object that would've been created otherwise (to let controller know how it was encrypted). More details here: https://github.com/bitnami-labs/sealed-secrets/blob/main/README.md#raw-mode-experimental